### ISSUE TITLE
内存泄漏问题

### ISSUE THREAD

**Issue by WWX1999** [actiontech/dble#3614]
dble版本3.21.10.3 ，dble.log刷
```
 Please pay attention to whether it is a memory leak. The maximum size of the DirectByteBufferPool that can be allocated at one time is 2097152, and the size that you would like to allocate is 32767
```
看了一下release记录，都有提到堆外内存泄漏，请问是否有一个比较稳定的版本

**Comment by LUAgam** [comment]
some memory leak issues were fixed before 3.21.10.12, you can consider upgrading to version [3.21.10.12](https://github.com/actiontech/dble/releases/tag/3.21.10.12%2Ftag)

**Comment by WWX1999** [comment]
thanks
